pandas/io/json/_normalize.py

```diff
@@ -26,12 +27,44 @@ def convert_to_line_delimits(s):
     return convert_json_to_lines(s)
 
 
+def _parse_use_keys(use_keys: Union[str, List, Callable, None]) -> Callable:
```

Comment:
```suggestion
def _parse_use_keys(use_keys: Optional[Union[str, List, Callable]]) -> Callable:
```

Typically prefer `Optional` over `Union[..., None]`